docs/src/examples/Cloudy_example.md

```diff
@@ -0,0 +1,518 @@
+
+Cloudy is used here in a "perfect model" (aka "known truth") setting, which
+means that the "observations" are generated by Cloudy itself, by running it with
+the true parameter values---in more realistic applications, one would use actual
+measurements of cloud properties.
+
+The schematic below gives an overview of the example: The input to the CES
+algorithm consists of data ``y``, the observational covariance ``Γ``, and prior
+parameter distributions. The data, a vector of moments of the droplet mass
+distribution, are obtained by running Cloudy with the parameters set to their
+true values. The covariance is obtained from model output. The calibration stage
+is performed by an ensemble Kalman inversion (EKI), in which Cloudy has to be
+run once per iteration and for each ensemble member. The resulting input-output
+pairs ``\{\theta_i, \mathcal{G}(\theta_i)\}_i`` are used to train an emulator
+model. This emulator ``\widetilde{\mathcal{G}}(\theta)`` is cheap to evaluate; it replaces the original parameter-to-data map in the Markov chain Monte Carlo (MCMC) sampling, which produces (approximate) samples of the posterior parameter distribution. These samples are the final output of the CES algorithm. 
+
+![Cloudy_overview](../assets/cloudy_ces_schematic.png)
+
+[This paper](https://agupubs.onlinelibrary.wiley.com/doi/full/10.1029/2022MS002994) describes Cloudy in much more detail and shows results of experiments using
+CES to learn model parameters.
+
+
+### Prerequisites
+
+In order to run this example, you need to install `Cloudy.jl`. The example was
+written using Cloudy v0.3.0, which you can install as follows:
+```julia
+pkg > add https://github.com/CliMA/Cloudy.jl.git#v0.3.0
+```
+
+### Running the example
+
+`Cloudy_calibrate.jl` performs the calibration using ensemble Kalman
+inversion; `Cloudy_emulate_sample.jl` fits an emulator and uses it to sample the
+posterior distributions of the parameters.
+Once Cloudy is installed, the example can be run from the julia REPL:
+```julia
+include("Cloudy_calibrate.jl")
+include("Cloudy_emulate_sample.jl")
+```
+
+### Walkthrough of the code: `Cloudy_calibrate.jl`
+
+This file performs the calibration stage of CES.
+
+#### Import packagages and modules
+
+First we load standard packages,
+
+```julia
+using Distributions
+using StatsBase
+using LinearAlgebra
+using StatsPlots
+using Plots
+using Plots.PlotMeasures
+using Random
+using JLD2
+```
+
+the Cloudy modules,
+
+```julia
+using Cloudy
+using Cloudy.ParticleDistributions
+using Cloudy.KernelTensors
+
+# Import the module that runs Cloudy
+include(joinpath(@__DIR__, "DynamicalModel.jl"))
+```
+
+and finally the EKP packages.
+
+```julia
+using EnsembleKalmanProcesses
+using EnsembleKalmanProcesses.Observations
+using EnsembleKalmanProcesses.ParameterDistributions
+using EnsembleKalmanProcesses.DataContainers
```

Comment:
this isn't in the project toml, should it be?

Reply:
Yes, good point, I added that too.